src/find_ice_labels.jl

```diff
@@ -84,3 +84,172 @@ function find_ice_labels(
     # @info "Done with ice labels" # to uncomment when logger is added
     return ice_labels
 end
+
+function get_image_peaks(arr, imgtype="uint8")
+    _, heights = imhist(arr, imgtype)
+
+    locs, heights, _ = Peaks.findmaxima(heights)
+
+    # TODO: make this conditional on input args
+    order = sortperm(heights; rev=true)
+    locs, heights = locs[order], heights[order]
+
+    return (locs=locs, heights=heights)
+end
+
+
+function get_ice_labels_mask(ref_img::Matrix{RGB{N0f8}}, thresholds, factor=1)
+    cv = channelview(ref_img)
+    cv = [float64.(cv[i, :, :]) .* factor for i in 1:3]
+    mask_ice_band_7 = cv[1] .< thresholds[1]
+    mask_ice_band_2 = cv[2] .> thresholds[2]
+    mask_ice_band_1 = cv[3] .> thresholds[3]
+    mask = mask_ice_band_7 .* mask_ice_band_2 .* mask_ice_band_1
+    @debug "Found $(sum(mask)) ice pixels"
+    return mask
+end
+
+# TODO: add test
```

Comment:
delete line 112, it seems like `test/test-tiled-ice-labels.jl` covers this script?